pages/snapshots/[id].js

```diff
@@ -1,14 +1,14 @@
 import { useState, useEffect } from 'react';
 import useSnapshot from 'lib/api/utils/useSnapshot';
-import { requestSnapshot, requestResources } from 'lib/api';
+import { requestSnapshot, requestResources, requestPrompts } from 'lib/api';
```

Comment:
Was this for testing as well? I don't expect to use the prompts inside this component yet, I don't think.

Reply:
ok good shout, I did use this for testing but ill remove them now